app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
+        <p class="govuk-body">An example is the <%= link_to "Driver and Vehicle Licensing Agency", "/government/organisations/driver-and-vehicle-licensing-agency", class: "govuk-link" %> (overseen by the <%= link_to "Department for Transport", "/government/organisations/department-for-transport", class: "govuk-link" %>).</p>
 
-        <div class="one-third">
-          <div class="inner-block">
-          </div>
-        </div>
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Work for us",
-              heading_level: 4,
-              margin_bottom: 2,
-              font_size: 19,
-            } %>
-
-            <p class="govuk-body"><%= link_to "Find and apply", "https://civilservicejobs.service.gov.uk/", class: "govuk-link" %> for vacancies in departments, executive agencies and non-departmental public bodies.</p>
-          </div>
-        </div>
-        <div class="one-third">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "Work with us",
-              heading_level: 4,
-              margin_bottom: 2,
-              font_size: 19,
-            } %>
-
-            <p class="govuk-body"><%= link_to "Search Contracts Finder", "/contracts-finder/", class: "govuk-link" %> for any government contract over £10,000 and get details of all previous tenders.</p>
-          </div>
-        </div>
-      </div>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Other public bodies",
+          font_size: "m",
+          heading_level: 3,
+          margin_bottom: 4,
+        } %>
 
-      <hr class="keyline">
+        <p class="govuk-body">These have varying degrees of independence but are directly accountable to ministers. There are 4 types of non-departmental public bodies (NDPBs).</p>
+        <p class="govuk-body">Executive NDPBs do work for the government in specific areas - for example, the Environment Agency.</p>
+        <p class="govuk-body">Advisory NDPBs provide independent, expert advice to ministers - for example, the Committee on Standards in Public Life.</p>
+        <p class="govuk-body">Tribunal NDPBs are part of the justice system and have jurisdiction over a specific area of law - for example, the Competition Appeal Tribunal.</p>
+        <p class="govuk-body">Independent monitoring boards are responsible for the running of prisons and treatment of prisoners - for example, Her Majesty's Inspectorate of Prisons.</p>
+
+         <ul class="govuk-list">
+          <li class="feature-value">
+            <a href="<%= organisations_path(anchor: 'departments') %>" class="feature-value__link">
+              <span class="feature-value__count"><%= @ministerial_department_count %></span>
```

Comment:
Ah this can be swapped for the newly created ["big numbers"](https://components.publishing.service.gov.uk/component-guide/big_number)

Reply:
Be careful doing this - we previously had to implement code to prevent it overlapping other stuff when users either zoomed in or increased their font size (I don't remember the exact scenario). Test thoroughly for this.

Reply:
*nods* - [PR Ref FYI ](https://github.com/alphagov/whitehall/pull/5900)

Reply:
Provided empty) alt text for the following images -
```
<%= image_tag @prime_minister.image_url(:s465), { alt: '' } %>
<%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Churchill_01.jpg', alt: '' %>
```

Reply:
Okay i've now updated the page to use the big number component. This was a little tricky due to the font size of the number in the component which has meant design changes following a review with Nik.

Zoomed in this looks fine and I can't see any overflow problems on this page. 

Attached a couple of screenshots.

_Infographic 1_

<img src="https://user-images.githubusercontent.com/87758239/134204273-b9c838be-5f06-43af-a2c4-9a6ecf35f88d.png" width="600">

_Infographic 2_

<img src="https://user-images.githubusercontent.com/87758239/134204234-e7f32dfc-c9df-45e4-ab7e-5883713523f9.png" width="600">